rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -1136,41 +1142,72 @@ private void testPegoutMinimumWithFeeVerification(Coin feePerKB, Coin value, boo
         List<LogInfo> logInfo = new ArrayList<>();
         BridgeEventLoggerImpl eventLogger = spy(new BridgeEventLoggerImpl(bridgeConstants, activationMock, logInfo, signatureCache));
         bridgeSupport = initBridgeSupport(eventLogger, activationMock);
+        when(feePerKbSupport.getFeePerKb()).thenReturn(feePerKB);
+
+        int pegoutSize = BridgeUtils.getRegularPegoutTxSize(activationMock, provider.getNewFederation());
+        Coin minValueAccordingToFee = feePerKbSupport.getFeePerKb().div(1000).times(pegoutSize);
+        Coin minValueWithGapAboveFee = minValueAccordingToFee.add(minValueAccordingToFee.times(bridgeConstants.getMinimumPegoutValuePercentageToReceiveAfterFee()).div(100));
+
+        Assertions.assertFalse(value.isLessThan(minValueWithGapAboveFee) ||
+            value.isLessThan(bridgeConstants.getMinimumPegoutTxValue()));
+
+        bridgeSupport.releaseBtc(buildReleaseRskTx(value));
+
+        Transaction rskTx = buildUpdateTx();
+        rskTx.sign(SENDER.getPrivKeyBytes());
+
+        verify(repository, never()).transfer(any(), any(), any());
+
+        Assertions.assertEquals(1, provider.getReleaseRequestQueue().getEntries().size());
+
+        Assertions.assertEquals(1, logInfo.size());
+        verify(eventLogger, times(1)).logReleaseBtcRequestReceived(any(), any(), any());
+        ArgumentCaptor<RejectedPegoutReason> argumentCaptor = ArgumentCaptor.forClass(RejectedPegoutReason.class);
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), argumentCaptor.capture());
```

Comment:
The `verify` statement checks that `logReleaseBtcRequestRejected` was never called, which makes capturing the argument redundant. 
```suggestion
        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
```